examples/AdminClient/Program.cs

```diff
@@ -112,6 +309,15 @@ public static async Task Main(string[] args)
                 case "create-topic":
                     await CreateTopicAsync(args[0], args[2]);
```

Comment:
maybe change this to pass commandArgs for consistency with the below (and it's nicer).